Gravity/Gravity/DAL/SQL/SqlDao.cs

```diff
@@ -1,41 +1,34 @@
 ﻿using System;
 using System.Collections.Generic;
 using Gravity.Base;
-using Gravity.Globals;
 using Gravity.Utils;
 using Relativity.API;
 
 namespace Gravity.DAL.SQL
 {
 	public partial class SqlDao : IGravityDao
 	{
-		private const int DefaultBatchSize = 1000;
+		private const int defaultBatchSize = 1000;
 
 		protected int workspaceId;
-		protected IHelper helper;
 		protected IDBContext dbContext;
 		protected IDBContext masterDbContext;
 		protected int batchSize;
 
 		private InvokeWithRetryService invokeWithRetryService;
 
+		private SqlDao(IDBContext workspaceContext, IDBContext masterContext)
```

Comment:
[Should-fix] This overload also needs the `InvokeWithRetrySettings` constructor. But if you're not using it yet, you could always just remove the whole thing.